vue.config.js

```diff
@@ -20,7 +22,17 @@ module.exports = {
                     "nosort": false,
                     "prefixFileName": true,
                 }
-            })
+            }),
+            new CopyWebpackPlugin([
+                {
+                  from: './README.md',
+                  to: './assets/'
+                },
+                {
+                    from: './src/**/*.md',
```

Comment:
Markdown files should not be in src folder, so this can be removed and `SLR.md` should be moved to the project root.